packages/react/src/patterns/sub-patterns/LinkList/__stories__/index.scss

```diff
@@ -0,0 +1,6 @@
+@import '../../../../../../styles/scss/patterns/sub-patterns/link-list/link-list';
+
+.#{$prefix}--link--list-story .#{$prefix}--link--list__list {
+  height: 500px;
```

Comment:
@mkothur @photodow @oliviaflory @larahanlon2 is this supposed to have a fixed height?

Reply:
@jeffchew the specs have a minimum height of 116px in all breakpoints **except the medium breakpoint**, but I don't see a fixed height anywhere.